verilog/transform/obfuscate.cc

```diff
@@ -47,9 +47,12 @@ static void ObfuscateVerilogCodeInternal(absl::string_view content,
       case verilog_tokentype::PP_Identifier:
         *output << (*subst)(token.text);
         break;
+        // System calls fail if encrypted
```

Comment:
suggest: "Preserve all $ID calls, including system task/function calls, and VPI calls"